code/__DEFINES/spans.dm

```diff
@@ -125,6 +125,9 @@
 #define SPAN_YELL(str) ("<span class='yell'>" + str + "</span>")
 #define SPAN_YELLOWTEAMRADIO(str) ("<span class='yellowteamradio'>" + str + "</span>")
 
+#define MATRIX_DANGER(X) (SPAN_DANGER(X))
+#define MATRIX_NOTICE(X) (SPAN_NOTICE(X))
```

Comment:
Why.

Reply:
Good question. Was from the direct port. Now that I get how those work, I can get rid of them for the normal ones.

Reply:
Please use the normal ones for now yea.

Reply:
Aaaand fixed